examples/experimental/i3s-17-and-debug/app-debug.js

```diff
@@ -163,7 +163,7 @@ export default class App extends PureComponent {
       selectedMapStyle: INITIAL_MAP_STYLE,
       debugOptions: INITIAL_DEBUG_OPTIONS_STATE,
       normalsDebugData: [],
-      normalsGap: DEFAULT_NORMALS_GAP,
+      normalsPercentage: DEFAULT_NORMALS_PERCENTAGE,
```

Comment:
Comment? Percentage of what? Total number of normals?

Reply:
Fixed. It is a percentage of triangles to show normals for.